book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] please ensure that Getters and Setters for `@rentals` should be present

Reply:
Hi @tufoinnkuo10  Thanks for the reminder, It's done. 😄